### DISCUSSION TITLE
[M-Z] (20 Gennaio 2020) Esercizio 9

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#478] [Esami & Esoneri M-Z]
### Traccia

<img width="476" alt="image" src="https://github.com/user-attachments/assets/989f4670-ec05-4abc-afdb-161dc0bd90c7">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200120.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200120.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/c9d5e093-9b68-40b6-bb33-ff76ebe20ac9)

## Tableau

![image](https://github.com/user-attachments/assets/51846828-f962-455a-b6c9-018afebf462d)

![image](https://github.com/user-attachments/assets/b04ed14d-027b-4105-9cb0-7da4e501f989)
